### PR TITLE
More updates to v1.0

### PR DESCRIPTION
**Pull request by erezsh**
Hopefully it's set up right this time.

### CODE REVIEW

**Comment by erezsh** [comment]
The Python3 grammar has been updated for compatibility with Lark.js

File: lark/exceptions.py [hunk 1]
```diff
@@ -18,7 +17,7 @@ class ConfigurationError(LarkError, ValueError):
     pass
 
 
-def assert_config(value, options, msg='Got %r, expected one of %s'):
+def assert_config(value, options: Sequence, msg='Got %r, expected one of %s'):
```
Comment (MegaIng) [inline lark/exceptions.py:20]: I think `Collection` is more correct here. a `set` is ok.

File: lark/indenter.py [hunk 2]
```diff
@@ -75,31 +74,31 @@ def always_accept(self):
     @property
     @abstractmethod
     def NL_type(self) -> str:
-        ...
+        return NotImplemented
```
Comment (MegaIng) [inline lark/indenter.py:77]: `raise NotImplementedError`

File: lark/parser_frontends.py [hunk 3]
```diff
@@ -32,20 +32,13 @@ def __init__(self, parser_type, lexer_type):
...
-        return ParsingFrontend(lexer_conf, parser_conf, options)
-
     def deserialize(self, data, memo, lexer_conf, callbacks, options):
         parser_conf = ParserConf.deserialize(data['parser_conf'], memo)
         parser = LALR_Parser.deserialize(data['parser'], memo, callbacks, options.debug)
         parser_conf.callbacks = callbacks
         return ParsingFrontend(lexer_conf, parser_conf, options, parser=parser)
 
-
+    # ... Continued later in the module
```
Comment (MegaIng) [inline lark/parser_frontends.py:41]: Why?

File: lark/tree.py [hunk 4]
```diff
@@ -18,6 +16,9 @@
     else:
         from typing_extensions import Literal
 
+###{standalone
+from collections import OrderedDict
```
Comment (MegaIng) [inline lark/tree.py:20]: Wasn't the goal to move all imports into `standalone.py`?

**Review by MegaIng — CHANGES_REQUESTED** [review]

File: lark/parser_frontends.py (hunk 3 above)
Comment (erezsh) [reply lark/parser_frontends.py:41]: So it won't be part of the standalone code.

File: lark/tree.py (hunk 4 above)
Comment (erezsh) [reply lark/tree.py:20]: Not exactly, I was mostly concerned about repetitive imports.

OrderedDict is only used once (and it a little meaningless in Python3, I just didn't think removing it fits into this PR)

File: lark/exceptions.py (hunk 1 above)
Comment (erezsh) [reply lark/exceptions.py:20]: You're right.

File: lark/parser_frontends.py (hunk 3 above)
Comment (MegaIng) [reply lark/parser_frontends.py:41]: I would suggest not splitting up the class. Instead move the complete class further down.

File: lark/tree.py (hunk 4 above)
Comment (MegaIng) [reply lark/tree.py:20]: Fair enough. (Also `OrderedDict` is still needed for `3.6` on non-CPython.)

File: lark/parser_frontends.py (hunk 3 above)
Comment (erezsh) [reply lark/parser_frontends.py:41]: It's also a bit strange to split the class in the middle. It breaks the indentation, and obfuscates the end of the standalone directive.

The only other place in Lark that happens is `tree.Tree`, and I'm not sure that it should stay that way.

But my mind isn't made up. If you think splitting the class using inheritance is too weird, I'll accept it.

**Pull request merged**